packages/components/src/heading/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './component';
+export * from './hook';
```

Comment:
This catch-all export made me think. What do we really expose from there? It doesn't seem to be used elsewhere.